src/Appwrite/Utopia/Database/Validator/Queries/Base.php

```diff
@@ -24,7 +24,9 @@ class Base extends Queries
      */
     public function __construct(string $collection, array $allowedAttributes)
     {
-        $collection = Config::getParam('collections', [])[$collection];
+        $config = Config::getParam('collections', []);
```

Comment:
What exactly are we validating here?

Reply:
Query allowed attributes, validating the attribute is valid in the collection structure.